index.html

```diff
@@ -462,13 +464,45 @@
       </ol>
     </section>
   </section>
+  <section>
+    <h2>Dependencies</h2>
+    <p>This specification depends on the following interfaces, attributes,
+    concepts, and terms which are defined in their linked specifications.</p>
+    <ul>
+      <li><code><dfn data-cite=
+      "!hr-time#idl-def-Performance">Performance</dfn></code> interface</li>
```

Comment:
How do we link to hr-time-2? Will !hr-time-2 do the trick? It's not clear how these urls are sourced..